lib/signaling/v2/peerconnection.js

```diff
@@ -1717,6 +1717,14 @@ function updateEncodingParameters(pcv2) {
       params.encodings[0].networkPriority = 'high';
     }
 
+    if (!isFirefox && sender.track.kind === 'video') {
+      let scale = 1;
+      params.encodings.forEach(encoding => {
+        encoding.scaleResolutionDownBy = scale;
+        scale *= 2;
+      });
+    }
```

Comment:
```suggestion

  // NOTE(mpatwardhan): All the simulcast encodings in Safari have
  // the same resolution. So, we make sure that the lower layers have
  // lower resolution, as seen in Chrome.
   params.encodings.forEach((encoding, i) =>
     encoding.scaleResolutionDownBy = 1 << (encodings.length - i - 1));
```

Reply:
@manjeshbhargav, I have incorporated your feedback. But instead of 
``` 
encoding.scaleResolutionDownBy = 1 << (encodings.length - i - 1));
```

I preferred 
```
encoding.scaleResolutionDownBy = 1 << i;
```

or is there  any reason to keep the 0th index as most scaled down one?

Reply:
@makarandp0 The first layer is usually the lowest resolution layer (Chromium has this behavior).